app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -201,23 +236,54 @@ class ChooseStep extends React.Component {
     let showNoResultsMessage = false;
     let isLoadingSearchOptions = false;
 
-    // Set different props for the dropdown depending on the field type.
-    switch (field.type) {
-      case "file_format":
+    // Handle rendering conditional fields.
+
+    // For the file format field, render a placeholder. This is a special case.
+    const fileFormatConditionalField = CONDITIONAL_FIELDS[0];
+    if (
+      field.type === fileFormatConditionalField.field &&
+      downloadType.type === fileFormatConditionalField.downloadType &&
+      !fileFormatConditionalField.triggerValues.includes(
```

Comment:
maybe there could be a helper function that takes in `fileFormatConditionalField` / `conditionalField`  to evaluate these conditionals since it's a little repetitive

Reply:
Did a fair amount of refactoring. Even simplified one of the props (`selectedFields`).